testdata/project-v3-config/controllers/suite_test.go

```diff
@@ -44,9 +43,7 @@ var testEnv *envtest.Environment
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
```

Comment:
Why we are changing the default scaffold more than just the import?

Reply:
https://onsi.github.io/ginkgo/MIGRATING_TO_V2#migration-strategy-2

Reply:
So, how could we achieve the same with the v2? Are we able to provide it via the scaffolds? How will it impact the author's projects? 

Also, it shows a breaking change for authors' projects so the emoji needs to be changed to `warning` :

Reply:
@haoxins could you please address this one ?